src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -111,29 +109,32 @@ <h5><span jhiTranslate="artemisApp.studentExamDetail.exercises">Exercises</span>
                             {{ exercise?.title }}
                         </td>
                         <td class="align-middle">
-                            <!-- TODO show result string -->
+                            <ng-container *ngIf="!!(exercise?.studentParticipations)[0].results">
+                                {{ (((exercise?.studentParticipations)[0]?.results)[0]?.score * exercise?.maxScore) / 100 }}
+                            </ng-container>
+                        </td>
+                        <td class="align.middle">
+                            {{ exercise?.maxScore }}
+                        </td>
+                        <td class="align.middle">
```

Comment:
```suggestion
                        <td class="align-middle">
```
See the comment above